okhttp-testing-support/src/test/kotlin/okhttp3/OkHttpClientTestRuleTest.kt

```diff
@@ -46,7 +47,7 @@ class OkHttpClientTestRuleTest {
       fail("")
     } catch (expected: AssertionError) {
       assertThat(expected).hasMessage("uncaught exception thrown during test")
-      assertThat(expected.cause).hasMessage("boom!")
+      assertThat(expected.cause!!).hasMessage("boom!")
```

Comment:
surprised there isnt a nice way to do this with clean messages not NPE.

Reply:
Ahh, it's Throwable, not string.  https://github.com/willowtreeapps/assertk/blob/5ff8a09f63543a18365dee1bb48dc9c4684aab9e/assertk/src/commonMain/kotlin/assertk/assertions/throwable.kt

Reply:
It would be with a `isNotNull()` which refines an `Assert<T?>` into an `Assert<T>`